### ISSUE TITLE
Open directory without any prompting.

### ISSUE THREAD

**Issue by NicholasBHubbard** [karthink/consult-dir#15]
When I select a directory from `consult-dir` I want to immediately open a dired buffer in the selected directory without any prompting. The implementation of `consult-dir` however seems to be based on the user getting a minibuffer prompt with their selection entered into the minibuffer by a call to `insert`.

I came up with this simple hack that gives the behavior I wanted by taking advantage of the fact that `consult-dir` uses the dynamic variable `default-directory`.

```
(defun my/consult-dir-find-file ()
    (interactive)
    (find-file default-directory))

(setq consult-dir-default-command #'my/consult-dir-find-file)
```
This works but it is a hack that relies on dynamic scoping.

I think it may be worthwhile to provide a command like this in `consult-dir` but maybe one that is a bit more elegant. Either way I wanted to leave this here for anybody else that may be desiring this behavior.

**Comment by karthink** [comment]
See also #11. This appears to be a common request, so I'll consider adding this as a built-in option for `consult-dir-default-command`.

**Comment by karthink** [comment]
This feature is now included in consult-dir (6ee8df75). Check the customization options for `consult-dir-default-command`, or

```elisp
(setq consult-dir-default-command #'consult-dir-dired)
```

> I think it may be worthwhile to provide a command like this in consult-dir but maybe one that is a bit more elegant.

Using the dynamic scoping (of default-directory) is part of the design of consult-dir, so there's no way to provide a more elegant version of your suggestion without rewriting a larger chunk of it.
